app.js

```diff
@@ -48,22 +49,32 @@ function addClass(insertedDiv, letter, i) {
 }
 
 function gameSetup() {
+  addLetterListeners();
+  insertNextImg();
+  let rndPickedProverb = proverbs[getRandomInt(proverbs.length)];
+  sentence.innerHTML = encryptSentence(rndPickedProverb);
+}
+
+function addLetterListeners() {
   const letters = document.querySelectorAll('.letter-wrapper');
-  [...letters].forEach(item => item.addEventListener('click', function(item) {
-    let pickedLetter = item.target.innerHTML;
-    if (pickedLetter.charAt(0) === '<') {
-      letterCheck(pickedLetter.charAt(3));
-    } else {
-      letterCheck(pickedLetter);
-    }
+  [...letters].forEach(item => item.addEventListener('click', (event) => {
```

Comment:
if arrow function has only one argument, you don't need to wrapp this by parenthesis. In fact -> you have it here...
also, if you have only one expression in function, you don't need to add curly braces. so:
```
[...letters].forEach(item => item.addEventListener('click', event => letterCheck(item.id)));
```